docs/source/user.rst

```diff
@@ -0,0 +1,199 @@
+..  _user:
+
+Here the simple examples cover the basics of creating messaging applications using kiwipy over RabbitMQ.
```

Comment:
Perhaps move this line under the **** headline?

Reply:
Lines removed. This is just to tell you what I want to add to the docs.